components/hab/src/command/studio.rs

```diff
@@ -59,6 +65,23 @@ pub fn start(ui: &mut UI, args: Vec<OsString>) -> Result<()> {
     inner::start(ui, args)
 }
 
+/// Retrieves the
+pub fn image_identifier() -> String {
+    let version: Vec<&str> = VERSION.split("/").collect();
```

Comment:
No, it won't always contain the `/` but this will always do the right thing. I wrote a test below which tests the case where a slash is missing (it's only present in a release build)